src/SourceBuild/content/eng/join-verticals.proj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.Build.Traversal">
+
+    <PropertyGroup>
+      <TargetFramework>$(NetCurrent)</TargetFramework>
+    </PropertyGroup>
+  
+    <ItemGroup>
+      <ProjectReference Include="$(TasksDir)Microsoft.DotNet.UnifiedBuild.Tasks\Microsoft.DotNet.UnifiedBuild.Tasks.csproj" BuildInParallel="true" />
+    </ItemGroup>
+
+    <Target Name="BuildTasksProject">
+      <MSBuild Projects="$(TasksDir)Microsoft.DotNet.UnifiedBuild.Tasks\Microsoft.DotNet.UnifiedBuild.Tasks.csproj" Targets="Build" />
+    </Target>
```

Comment:
This target isn't necessary as the referenced msbuild project already gets built because of the `ProjectReference` item above.